src/commands/community/list/template.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as os from 'os';
+import { Messages } from '@salesforce/core';
+import { requiredOrgFlagWithDeprecations, SfCommand } from '@salesforce/sf-plugins-core';
+import { CommunityTemplatesResource } from '../../../shared/community/connect/CommunityTemplatesResource';
+import { ConnectExecutor } from '../../../shared/connect/services/ConnectExecutor';
+import { CommunityTemplatesListResponse } from '../../../shared/community/defs/CommunityTemplatesListResponse';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-community', 'template.list');
+
+/**
+ * A command to fetch available community templates a community. This is just an sfdx wrapper around
+ * the get available community templates connect endpoint
+ */
+export class CommunityListTemplatesCommand extends SfCommand<CommunityTemplatesListResponse> {
+  public static readonly aliases = ['force:community:template:list', 'community:template:list'];
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
```

Comment:
fyi, there's an upcoming linter rule about this pattern.   

https://github.com/salesforcecli/eslint-plugin-sf-plugin/pull/120/files#diff-9f23819fa6dc2571f2f8ab122a36b3ea436a4c1b1561801f0d110b680c25245bR53-R59